source/parameters/bend.md

```diff
@@ -0,0 +1,193 @@
+Also see `e1_rect` and `e2_rect`. The relationship is
+  ```{code} python
+  :label: eeaeea
+
+  e1 = e1_rect + angle/2 
+  e2 = e2_rect + angle/2
+  ```
+%
+- **e1_rect, e2_rect**
+Face angle rotations like `e1` and `e2` except angles are measured with respect to
+fiducial lines that are parallel to each other and rotated by `angle`/2 from the radial
+{math}`x_1` and {math}`x_2` axes as shown in {numref}`f:sbend`.
+Zero `e1_rect` and `e2_rect` gives a rectangular magnet shape.
+%
+- **edge_int1, edge_int2**
+The field integral for the entrance pole face `edge_int1` is given by
+  ```{math}
+  \text{edge_int1} = \int_{pole} \!\! ds \frac{B_y(s) (B_{y0} - B_y(s))}
+  {2 , B_{y0}^2}
+  ```
+  For the exit pole face there is a similar equation for `edge_int2`
+
+  Note: In Bmad and MAD, these integrals are represented by the product of `fint` and `hgap`.
+
+  `edge_int1` and `edge_int2` can be related to the Enge function which is sometimes used to model the
+  fringe field. The Enge function is of the form
+  ```{math}
+  B_y(s) = \frac{B_{y0}}{1 + \exp[P(s)]}
+  ```
+  ```{math}
+  P(s) = C_0 + C_1 , s + C_2 , s^2 + C_3 , s^3 + , \ldots
+  ```
+  The {math}`C_0` term simply shifts where the edge of the bend is. If all the {math}`C_n` are zero except for
+  {math}`C_0` and {math}`C_1` then
+  ```{math}
+  C_1 = \frac{1}{2 \cdot \text{field_int}}
+  ```
+%
+- **g, rho (output param)**
+The Reference bending radius which determines the reference coordinate system is `rho` (see
+[](#s:ref)). `g` = `1/rho` is the "bend strength" and is proportional to the Reference
+dipole magnetic field. `g` is related to the reference magnetic field `bend_field_ref` via
+  ```{math}
+  :label: gqpb
+
+  \text{g} = \frac{q}{p_0} \cdot \text{bend_field_ref}
+end{equation}
+where {math}`q` is the charge of the reference particle and {math}`p_0` is the reference momentum. It is
+important to keep in mind that changing `g` will change the Reference orbit ([](#s:coords.3)) and
+hence will move all downstream lattice elements in space.
+
+  The total bend strength felt by a particle is the vector sum of `g` plus the zeroth order
+magnetic multipole. If the multipole `tilt0` and `Ks0` is zero, the total bend strength is
+  ```{code} yaml
+  norm_bend_field = g + Kn0
+  ```
+  Changing the multipole strength `Kn0` or `Ks0` leaves the Reference orbit and the positions of
+all downstream lattice elements
+unchanged but will vary a particle's orbit. One common mistake when designing lattices is to vary
+`g` and not `Kn0` which results in downstream elements moving around. See Sref{s:ex.chicane}
+for an example.
+
+  Note: A positive `g`, which will bend particles and the reference orbit in the {math}`-x` direction
+represents a field of opposite sign as the field due a positive `hkick`.
+%
+- **h1, h2**
+The attributes `h1` and `h2` are the curvature of the entrance and exit pole faces.
+%
+- **L, L_arc, L_chord, L_sagitta (output param)** 
```

Comment:
By `L` here, do you mean the `Length` parameter?  It was stated earlier in this section that the `Length` parameter is defined outside any parameter group.  If we want to clarify how the length is defined for a Bend element, then I think this should be placed elsewhere, outside of the parameter group description.

Reply:
This should be `Length`. Fixed.